database/src/pool/postgres.rs

```diff
@@ -617,7 +622,7 @@ where
             pstat_series: self
                 .conn()
                 .query(
-                    "select id, crate, profile, scenario, backend, metric from pstat_series;",
+                    "select id, crate, profile, scenario, backend, metric, compiler_target, from pstat_series;",
```

Comment:
```suggestion
                    "select id, crate, profile, scenario, backend, metric, compiler_target from pstat_series;",
```
There was an extra comma here.